copy/copy.go

```diff
@@ -148,6 +148,11 @@ type Options struct {
 	// so that storage.ResolveReference returns exactly the created image.
 	// WARNING: It is unspecified whether the reference also contains a reference.Named element.
 	ReportResolvedReference *types.ImageReference
+
+	// SourceDateEpoch, if set, and if applicable for destination, will force
+	// timestamps to this value. Useful for oci-archive outputs where this will
+	// set the create/mod/access timestamps in each tar entry
+	SourceDateEpoch *time.Time
```

Comment:
I think something like this would accurately describe what the option does, WDYT?
```suggestion
	// DestinationTimestamp, if set, will force timestamps of content created in the destination to this value.
	// Most transports don't support this.
	//
	// In oci-archive: destinations, this will set the create/mod/access timestamps in each tar entry
	// (but not a timestamp of the created archive file).
	DestinationTimestamp *time.Time
```

Reply:
I think that looks good. I've changed as suggested and rebased/squashed.

Thanks for putting the thought in. Naming things is Hard.